.pre-commit-config.yaml

```diff
@@ -0,0 +1,35 @@
+default_stages: [commit]
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.1.0
+    hooks:
+      - id: check-yaml
+        args:
+          - --allow-multiple-documents
+      - id: check-toml
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+    files: '.*\.go$'
```

Comment:
This is incorrectly indented, is it meant to apply to the `trailing-whitespace` hook?